docs/_articles/en/codebuilder/cb-diffs.md

```diff
@@ -2,23 +2,18 @@
 title: What's Different
 lang: en
 ---
+
 ## What’s Different in Code Builder
 
 Code Builder unleashes the power of Salesforce Extensions for VS Code and offers the same rich features as the desktop version. No matter whether you're using VS Code on your desktop or Code Builder from a browser, the Salesforce extensions you access are the same. While the extensions generally function the same way regardless of how they're accessed, there are some nuances to be aware of:
 
-* Code Builder is installed as a managed package, and it comes with everything you need: VS Code, Salesforce extensions, and Salesforce CLI. No installation required. 
-* Unlike VS Code for the desktop, you can't add any new extensions to your Code Builder development environment.
-* Because Code Builder is inherently lightweight and optimized to run in the cloud, access to certain functionalities is limited:    
-    * Functions development is available only for Javascript and Typescript functions.
-    * LWC Local Development (beta) is not available in Code Builder. You can't preview LWC components locally, but can deploy them to your org to view them.
-* With Code Builder, you can work from anywhere. Bookmark your development environment to return to it, or access the link from a different machine. 
-* Unlike a desktop IDE, Code Builder saves your work for you without you explicitly having to do so. However, your beta environment's usage is capped at 20 hours. Once the clock runs out, your data and metadata are deleted. Be sure to push your work to your source control system or deploy your changes to your org. 
-* Because your Code Builder instance runs in the cloud, authorizing an org from it follows a different (device) flow than the desktop IDE. See [Connect to Different Org](https://developer.salesforce.com/tools/vscode/en/codebuilder/cb-start/#connect-to-a-different-org) for steps.
+- Because your Code Builder instance runs in the cloud, authorizing an org from it follows a different (device) flow than the desktop IDE. See [Connect to Different Org](https://developer.salesforce.com/tools/vscode/en/codebuilder/cb-start/#connect-to-a-different-org) for steps.
+- Code Builder is installed as a managed package, and it comes with everything you need: VS Code, Salesforce extensions, and Salesforce CLI. No installation required on your local machine.
+- Unlike VS Code for desktop, you cannot access the Microsoft Marketplace from Code Builder. Instead, you can add extensions available in [Open VSX](https://open-vsx.org/) marketplace to your Code Builder environment. This means that you might see differences in the extensions that are available.
+- Third party authorization sequence is different in Code Builder.
```

Comment:
```suggestion
- The third-party authorization sequence is different in Code Builder.
```

Reply:
3rd party auth seems a little abstract to me - can you simplify? Like: The steps you take to authorize environments in Code Builder is a bit different than the desktop version.